doc/howto/server_expose.md

```diff
@@ -1,15 +1,39 @@
 (server-expose)=
 # How to expose LXD to the network
 
-By default, LXD can be used only by local users through a Unix socket.
+By default, LXD can be used only by local users through a Unix socket and is not accessible over the network.
+
+To expose LXD to the network, you must configure it to listen to addresses other than the local Unix socket.
+To do so, set the [`core.https_address`](server) server configuration option.
 
-To expose LXD to the network, set the [`core.https_address`](server) server configuration option.
 For example, to allow access to the LXD server on port `8443`, enter the following command:
 
     lxc config set core.https_address :8443
 
+To allow access through a specific IP address, use `ip addr` to find an available address and then set it.
+For example:
+
+```{terminal}
+:input: ip addr
+
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+2: lxdbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
+    link/ether 00:16:3e:8d:f3:72 brd ff:ff:ff:ff:ff:ff
+    inet 10.64.82.1/24 scope global lxdbr0
+       valid_lft forever preferred_lft forever
+    inet6 fd42:f4ab:4399:e6eb::1/64 scope global
+       valid_lft forever preferred_lft forever
+:input: lxc config set core.https_address 10.64.82.1
```

Comment:
This is a local address, so maybe not that helpful? 
I'd like to add an example output so people know what to look out for, but is that "real-life" enough?

Reply:
Can you show the output of `ip addr` where it shows lo, lxdbr0 and another external interface, such as eth0, enp5s0 etc. Then show using the IP address from that external interface.

Reply:
spot on.